tensorflow/python/training/monitored_session.py

```diff
@@ -339,6 +345,14 @@ def MonitoredTrainingSession(master='',  # pylint: disable=invalid-name
   elif save_summaries_steps == USE_DEFAULT:
     save_summaries_steps = None
 
+  if save_checkpoint_steps == USE_DEFAULT and save_checkpoint_secs == USE_DEFAULT:
+    save_checkpoint_steps = 5000
```

Comment:
this is a behavior change. Please do not change it. set secs to 600 for default case.